app/mock/users.ts

```diff
@@ -0,0 +1,42 @@
+import type { User } from '@prisma/client';
+
+const mockUsers: User[] = [
```

Comment:
Any reason we can't use the users in the seed?

Reply:
We don't need to mock anything, use the database please.